packages/contracts-bedrock/contracts/echidna/FuzzAddressAliasing.sol

```diff
@@ -24,7 +24,10 @@ contract EchidnaFuzzAddressAliasing {
     }
 
     /**
-     * @notice Verifies that testRoundTrip(...) did not ever fail.
+     * INVARIANT: Address aliases are always able to be undone.
```

Comment:
Perhaps we could use a custom tag here? `@custom:invariant`

Like this:
https://github.com/ethereum-optimism/optimism/blob/f7f6d76f882ec396c3d4ce89b2fc9cc26e7fc324/packages/contracts-bedrock/contracts/legacy/L1BlockNumber.sol#L11